assets/src/edit-story/components/library/panes/text/textSets/textSetsPane.js

```diff
@@ -92,9 +92,23 @@ function TextSetsPane({ paneRef }) {
 
   const addIdToTextSets = useCallback(
     (textSetsArray = []) =>
-      textSetsArray.map((elements) => {
+      textSetsArray.map(({ textSetFonts = [], textSetCategory, elements }) => {
         return {
           id: `text_set_${uuidv4()}`,
+          title: sprintf(
```

Comment:
Nit: I'd imagine a field named `title` would be visible. `ariaLabel`?

Reply:
yeah i thought about it, we're using `title` on page templates for this same structure though and it's really a title of the text set that doesn't need to just be the ariaLabel. I don't really care one way or other if you feel strongly about this rename.